update docs for waf policy version name updates (#2039)
nginx/documentation: content/nginx-one-console/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## June 15, 2026

### F5 WAF for NGINX: Updated policy version names

> The names of policy versions will no longer use the version's created date as a name, and will now be numbered sequentially (v1, v2, v3, etc). Policy versions may also be given an optional version comment for later reference. Existing policy versions have been migrated to use the new naming scheme in place of the previous one.

Reviewer: ```suggestion
Policy versions are now numbered sequentially (v1, v2, v3, etc.) instead of using the creation date as the version name. You can also add an optional comment to each version for reference. Existing policy versions now use the new naming scheme.
```

Edits for sentence length, active voice. See: https://github.com/F5Docs/style-guide
